src/main/java/io/vertx/ext/couchdb/CouchdbClient.java

```diff
@@ -46,68 +61,215 @@ static CouchdbClient create(Vertx vertx, WebClient client, Credentials credentia
   }
 
   /**
-   * CouchDB server status as JSON object.
+   * CouchDB server instance status as JSON object.
    *
+   * @see https://docs.couchdb.org/en/stable/api/server/common.html#/
    * @return Future with the status of the CouchDB server, fails with
-   * {@link CouchdbException} if the operation fails.
+   *         {@link CouchdbException} if the operation fails.
    */
   Future<JsonObject> status();
 
-  Future<JsonArray> activeTasks();
+  /**
+   * CouchDB session status as JSON object.
+   *
+   * @see https://docs.couchdb.org/en/stable/api/server/common.html#/
+   * @return Future with the status of the CouchDB server, fails with
+   *         {@link CouchdbException} if the operation fails.
+   */
+  Future<JsonObject> session();
 
   /**
-   * List all databases.
+   * request one or more uuids
    *
-   * @return Future with the list of databases, fails with {@link CouchdbException} if the operation fails.
+   * @see https://docs.couchdb.org/en/stable/api/server/common.html#_uuids
+   * @return Future with the status of the CouchDB server, fails with
+   *         {@link CouchdbException} if the operation fails.
    */
-  Future<JsonArray> allDbs();
+  Future<JsonObject> uuids(int count);
 
   /**
-   * List all databases, limited by the query parameters.
+   * Makes a call to the CouchDB server with the given parameters.
    *
-   * @param options JsonObject with the query parameters.
-   * @return Future with the list of databases, fails with {@link CouchdbException} if the operation fails.
+   * @param params JsonObject with the headers, path, method, parameters, and
+   *               payload (if any).
+   * @return Future with the result of the call, fails with
+   *         {@link CouchdbException} if the
+   *         operation fails.
    */
-  Future<JsonArray> allDbs(JsonObject options);
+  Future<Buffer> rawCall(JsonObject params);
 
   /**
-   * List all databases, limited by the query parameters.
+   * Retrieves a specified database.
    *
-   * @param options JsonObject with the query parameters.
-   * @return Future with the list of databases, fails with {@link CouchdbException} if the operation fails.
+   * @param databaseName The name of the database, must follow specific naming
+   *                     rules.
+   * @return Future with a CouchDbDatabase instance
    */
-  Future<JsonArray> dbsInfo(JsonObject options);
+  Future<CouchDbDatabase> getDatabase(String databaseName);
 
   /**
-   * Makes a call to the CouchDB server with the given parameters.
+   * @return the vertx instance
+   */
+  VertxInternal getVertx();
```

Comment:
Question Would `VertxInternal` be more beneficial for our users than returning `Vertx`?

Reply:
As the name implies: for library internal use only

Reply:
Removed in later PR